pkg/k8s/watchers/watcher.go

```diff
@@ -458,14 +459,6 @@ func (k *K8sWatcher) resourceGroups() (beforeNodeInitGroups, afterNodeInitGroups
 		k8sGroups = append(k8sGroups, resources.K8sAPIGroupSecretV1Core)
 	}
 
-	// To perform the service translation and have the BPF LB datapath
-	// with the right service -> backend (k8s endpoints) translation.
-	if k8s.SupportsEndpointSlice() {
-		k8sGroups = append(k8sGroups, resources.K8sAPIGroupEndpointSliceV1Beta1Discovery)
-	} else if k8s.SupportsEndpointSliceV1() {
-		k8sGroups = append(k8sGroups, resources.K8sAPIGroupEndpointSliceV1Discovery)
```

Comment:
Ah probably should default to `EndpointSliceV1` for this and the metrics instead of `EndpointV1Core` as above.